docs/source/installation.rst

```diff
@@ -16,29 +15,27 @@ For more information on installing collections, see `using collections`_.
 
 Ansible Galaxy
 --------------
-
-You can use the `ansible-galaxy`_ command with the option ``install`` to
-install a collection on your system (control node) hosted in Galaxy.
-
 Galaxy enables you to quickly configure your automation project with content
-from the Ansible community. Galaxy provides prepackaged units of work known as
-collections. If you have installed a prior version you may need to force
-overwriting an existing collection with the ``--force`` option.
+from the Ansible community.
 
-Here are examples for installing the **IBM z/OS core collection**:
+Galaxy provides prepackaged units of work known as collections. You can use the
+`ansible-galaxy`_ command with the option ``install`` to install a collection on
+your system (control node) hosted in Galaxy. If you have installed a prior
+version, you must overwrite an existing collection with the ``--force`` option.
+
+Here are a few examples of installing the **IBM z/OS core collection**:
 
 .. code-block:: sh
 
    $ ansible-galaxy collection install ibm.ibm_zos_core\
```

Comment:
@RadhaV2020 can you remove the trailing back slash from the sample:
`$ ansible-galaxy collection install ibm.ibm_zos_core\`

Reply:
will do.